migration/src/spreadsheets.rs

```diff
@@ -178,19 +180,20 @@ impl SheetResult {
                     document_short_name: doc_string,
                     id: None,
                     wordpress_id: wp_id,
+                    section: intro_or_body,
                 };
 
                 chapters.push(new_chapter);
             }
         }
-        let self_genre_chapters = chapters;
+        let self_body_chapters = chapters;
 
         Ok(Collection {
             title: self_title.to_string(),
             wordpress_menu_id: Some(*self_wordpress_menu_id),
             slug: self_slug.to_string(),
             intro_chapters: self_intro_chapters,
-            genre_chapters: self_genre_chapters,
+            body_chapters: self_body_chapters,
```

Comment:
Since we have a marker for this inside the chapter now, do you think it's reasonable to condense this down to one field, `chapters`?

Reply:
I agree, fixed this!

Reply:
am I all set to merge now that that's done?

Reply:
Yup, I already approved it for merging